src/tests/Equinor.ProCoSys.IPO.WebApi.IntegrationTests/Invitations/InvitationsControllerTests.cs

```diff
@@ -40,20 +40,309 @@ public async Task GetInvitationsByCommPkgNo_AsViewer_ShouldGetInvitations()
+                "InvitationForAcceptingTitle",
+                "InvitationForAcceptingDescription",
+                InvitationLocation,
+                DisciplineType.DP,
+                _invitationStartTime,
+                _invitationEndTime,
+                _participantsForSigning,
+                _mcPkgScope,
+                null
+            );
+
+            var invitation = await InvitationsControllerTestsHelper.GetInvitationAsync(
+                UserType.Viewer,
+                TestFactory.PlantWithAccess,
+                invitationToAcceptId);
+
+            var completerPerson = invitation.Participants
+                .Single(p => p.Organization == Organization.Contractor).Person;
+
+            if (completerPerson.Person.Id != null)
+            {
+                var completePunchOutDto = new CompletePunchOutDto()
+                {
+                    InvitationRowVersion = invitation.RowVersion,
+                    ParticipantRowVersion = completerPerson.Person.RowVersion,
+                    Participants = new List<ParticipantToChangeDto>
+                    {
+                        new ParticipantToChangeDto
+                        {
+                            Id = (int)completerPerson.Person.Id,
+                            Note = "Some note about the punch out round or attendee",
+                            RowVersion = completerPerson.Person.RowVersion,
+                            Attended = true
+                        }
+                    }
+                };
+
+                // Punch round must be completed before it can be accepted
+                await InvitationsControllerTestsHelper.CompletePunchOutAsync(
+                    UserType.Completer,
+                    TestFactory.PlantWithAccess,
+                    invitationToAcceptId,
+                    completePunchOutDto);
+
+                var accepterPerson = invitation.Participants
+                    .Single(p => p.Organization == Organization.ConstructionCompany).Person;
+
+                if (accepterPerson.Person.Id != null)
+                {
+                    var acceptPunchOutDto = new AcceptPunchOutDto
+                    {
+                        InvitationRowVersion = invitation.RowVersion,
+                        ParticipantRowVersion = accepterPerson.Person.RowVersion,
+                        Participants = new List<ParticipantToUpdateNoteDto>
+                        {
+                            new ParticipantToUpdateNoteDto
+                            {
+                                Id = (int)accepterPerson.Person.Id,
+                                Note = "Some note about the punch out round or attendee",
+                                RowVersion = accepterPerson.Person.RowVersion
+                            }
+                        }
+                    };
+
+                // Act
+                await InvitationsControllerTestsHelper.AcceptPunchOutAsync(
```

Comment:
why do we not need RowVersion as input here? The commandhandler creates a new RowVersion so I guess we should Assert for changed RowVersion here too?